### DISCUSSION TITLE
Preview Image Opacity / Blur

### DISCUSSION THREAD

**Discussion by philhk** [ppy/osu#26568] [Ideas] [answered]
### Here's a picture so we all know exactly what I'm talking about:
![Unbenannt](https://github.com/ppy/osu/assets/66958695/1f3bc74d-e26e-4833-ad21-2676e2856b42)
So... I would like a feature that allows me to change the opacity and blur of all beatmaps globally (the previews in the red boxes and the whole background). Why? Well, some osu! maps have rather "suggestive" backgrounds. Now, can you imagine what happens after playing osu! for 6 hours and encountering more than 30 of them? I get quite horny, and I'm sure I'm not the only one who feels this way. The thing is, it's really annoying. I just want to enjoy the game without getting the urge over time. I hope I'm not alone in this, but it seems like nobody is talking about it. When I searched for similar ideas, I couldn't find anything. Please don't think I'm weird.

**Comment by bdach** [comment, marked as answer]
https://github.com/ppy/osu/discussions/26481
https://github.com/ppy/osu/discussions/24892
https://github.com/ppy/osu/discussions/20950
https://github.com/ppy/osu/discussions/13333
